src/logger/MetricsLogger.ts

```diff
@@ -19,6 +19,14 @@ import { IEnvironment } from '../environment/IEnvironment';
 import { MetricsContext } from './MetricsContext';
 import { Unit } from './Unit';
 
+type Metrics = { [key: string]: number | number[]; };
```

Comment:
This type definition doesn't allow a developer to define the "unit" of the metric (e.g. count, milliseconds, etc.).

I was hoping to set something like:

```js
 {
    Name: 'ResponseTime',
    Value: 59,
    Unit: Unit.Milliseconds
}
```

Also, multiple metrics may apply to the same dimensions, shouldn't we define this as an array of metric objects as discussed https://github.com/awslabs/aws-embedded-metrics-node/issues/46?

Reply:
Yeah agree on unit. The idea here was that the key would be used as the metric name. I don’t recall exactly why I proposed it this way. Happy to go with this.